src/lib/builder/net-builder/net-builder.ts

```diff
@@ -52,7 +52,6 @@ export class NetBuilderClass implements NetBuilder {
         is_digital_signal: this.props.is_digital_signal,
         is_power: this.props.is_power,
         is_ground: this.props.is_ground,
-        trace_width: this.props.trace_width,
```

Comment:
nets should also have a trace width!

Reply:
basically there's a priority system, net comes last in priorities

Reply:
this is so you can say "the 5V net should generally have this traceWidth, but this hint should override that because this point gets hot..."

Reply:
Oops, yeah! this was giving type error and to fix that I removed it for a while. I think because of some version mismatch. I will add this back